src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -159,19 +173,36 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
-  uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
-  uint64_t end_ns = thread_state_slice->end_timestamp_ns();
 
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+
+  if (thread_state_slice->wakeup_tid() != orbit_base::kInvalidThreadId) {
+    std::string blocked_or_created = GetWakeupReason(thread_state_slice->wakeup_reason());
+    std::string thread_name = capture_data_->GetThreadName(thread_state_slice->wakeup_tid());
+    std::string process_name = capture_data_->GetThreadName(thread_state_slice->wakeup_pid());
+    tooltip += absl::StrFormat(
+        "<br/>"
+        "<b>Was %s By Process:</b> %s [%d]"
+        "<br/>"
+        "<b>Was %s By Thread:</b> %s [%d]",
```

Comment:
I don't think we should capitalize every word here: "Was created by process/thread"

Reply:
I thought these represent a header therefore I capitalized everthing.

Reply:
Hmm, I don't think these should be seen as headers. Also, in `GpuSubmissionTrack.cpp` and `GpuDebugMarkerTrack.cpp` we have "Submitted from process/thread" not capitalized, so we should be consistent.

Reply:
This should be done now.